src/Fl_PNG_Image.cxx

```diff
@@ -131,6 +132,9 @@ void Fl_PNG_Image::load_png_(const char *name_png, const unsigned char *buffer_p
       ld(ERR_FILE_ACCESS);
       return;
     }
+    if (offset > 0 && fseek(fp, (long)offset, SEEK_SET) == -1) {
+      rewind(fp);
+    }
```

Comment:
Should the function return here instead of calling `rewind()` and ignoring the `fseek()` error?